WikiWatershed/model-my-watershed
# Text Updates

## Overview

Brief description of what this PR does, and why it is needed.

Connects #1964 

### Demo

![image](https://user-images.githubusercontent.com/1430060/28400180-6c40d30e-6cdf-11e7-81c6-94be3c10ecf6.png)

![image](https://user-images.githubusercontent.com/1430060/28400186-7687a7de-6cdf-11e7-8eef-a29530becb98.png)

![image](https://user-images.githubusercontent.com/1430060/28400193-7fef617c-6cdf-11e7-9e59-090230072b30.png)

![image](https://user-images.githubusercontent.com/1430060/28400200-8f6cb528-6cdf-11e7-99bd-fec7a05cc544.png)

![image](https://user-images.githubusercontent.com/1430060/28400204-98317c34-6cdf-11e7-86be-f9cb779240c3.png)

![image](https://user-images.githubusercontent.com/1430060/28400215-a6b4cbf8-6cdf-11e7-9a89-123fd4122e21.png)

![image](https://user-images.githubusercontent.com/1430060/28400226-aefaea7c-6cdf-11e7-94c9-2ce8c4823be4.png)

![image](https://user-images.githubusercontent.com/1430060/28400244-cc80760c-6cdf-11e7-8d9a-f245b9499813.png)

### Notes

They wanted help links for each of the models in the model selection boxes in Analyze, in the last screenshot. I tried to add them, but clicking the links would also select the model and proceed. It may take some time to figure out how to do it right, so I skipped them for now. I did add them to `base.py` as `modelPackage.help_link` so we can add them in when we can.

## Testing Instructions

 * Check out this branch and run `update`
 * Ensure there are no more `Lorem ipsums` in the app
 * Ensure text lines up with specifications [here](https://docs.google.com/document/d/13HVyNkrtKKtLufBguCZJq0RDabBCTkjL0vT1PCwz8Lg/edit)
